### PR TITLE
feat: hook isMobile

### PR DESCRIPTION
**Pull request by juliahermak**

### CODE REVIEW

**Comment by juliahermak** [comment]
[Make isMobile() reactive](https://trello.com/c/niToak2R/510-make-ismobile-reactive)

**Review by bludnic — APPROVED** [review]

File: src/hooks/useIsMobile.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { detect } from 'detect-browser'
+import { ref, onMounted, onUnmounted } from 'vue'
+
+export function useIsMobile() {
+  const isMobile = ref(false)
+
+  function isMobileCheck() {
+    const browser = detect()
+    const isMobileDevice =
+      browser && browser.os ? ['android', 'ios'].includes(browser.os.toLowerCase()) : false
```
Comment (RealGoodProgrammer) [inline src/hooks/useIsMobile.ts:10]: Just a suggestion: add "blackberry" and "Windows Mobile". Some people still use it for security reasons.

Comment (juliahermak) [reply src/hooks/useIsMobile.ts]: Added

**Comment by bludnic** [comment]
Check the actual values for Windows Mobile and Blackberry @juliahermak

**Review by PaulDremanovich — APPROVED** [review]

**Comment by aineo** [comment]
[Fix Github Actions deprecations](https://trello.com/c/E2OeEsd2/541-fix-github-actions-deprecations)

**Pull request merged**